components/wb-fieldflow/fieldflow.js

```diff
@@ -829,7 +829,7 @@ var componentName = "wb-fieldflow",
 			}
 
 			if ( !itmLabel ) {
-				itmLabel = firstNode.nodeValue;
+				itmLabel = $( itmCached ).html();
```

Comment:
This matches what's being done for anchors at:
https://github.com/wet-boew/GCWeb/blob/74b26ebe2453008fb1c45b13f6562f2eef18bb9a/components/wb-fieldflow/fieldflow.js#L789

(I probably would've used the [``innerHTML`` property](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML) instead of [jQuery's ``html()`` method](https://api.jquery.com/html/) if there hadn't been any precedent)